src/theme/src/themes/v5/foundational-styles/tokens.js

```diff
@@ -267,6 +267,8 @@ const fills = {
 // Lifted from v5 scales
 const spinnerColor = 'rgba(67, 90, 111, 0.47)'
 const overlayBackgroundColor = 'rgba(67, 90, 111, 0.7)'
+const codeBackgroundColor = 'rgba(16, 112, 202, 0.06)'
+const codeBorderColor = 'rgba(16, 112, 202, 0.14)'
```

Comment:
The duplication here exists until the v5 theme can basically be a clean override / composition of the v6 theme (which we're getting close to doing). Until then, just want to avoid breaking anything.